src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -548,20 +548,8 @@ static int enumerator_scan_dir_and_add_devices(sd_device_enumerator *enumerator,
                         continue;
                 }
 
-                /*
```

Comment:
hm, this is quite a change from the status quo ante if i grok this properly. preivously we said that devices for which no props are set just don't exist in our view. With this change now they do. This will mean that on some systems the number of db entries will explode. i.e. some systems with ridiciulous amounts of NUMA memory slots have like a million deivces. so far we'd process those in parallel but not store anything about this. which made this OK. But if we now store a db entry for each things would become quite different.

What's the rationale for this? So far we said: if you want to use the `sd_device_is_initialzed()` to wor, make sure to have a rule on your devices that sets at least one prop. That way people could specifically select the devices they care about, and thus avoid the NUMA memory slot issue, since noone would set a rule for those...

Reply:
Yeah, this change make udevd may create huge number of database files. That's expected, and this is for making `sd_device_is_initialized()` work for all devices. But I did not consider the NUMA case. I will drop the commit.